client/deployments/client.go

```diff
@@ -32,13 +33,51 @@ import (
 	"github.com/mendersoftware/mender-cli/log"
 )
 
+type artifactsList struct {
+	artifacts []artifactData
+}
+
+type artifactData struct {
+	ID                    string   `json:"id"`
+	Description           string   `json:"description"`
+	Name                  string   `json:"name"`
+	DeviceTypesCompatible []string `json:"device_types_compatible"`
+	Info                  struct {
+		Format  string `json:"format"`
+		Version int    `json:"version"`
+	} `json:"info"`
+	Signed  bool `json:"signed"`
+	Updates []struct {
+		TypeInfo struct {
+			Type string `json:"type"`
+		} `json:"type_info"`
+		Files []struct {
+			Name     string    `json:"name"`
+			Checksum string    `json:"checksum"`
+			Size     int       `json:"size"`
+			Date     time.Time `json:"date"`
+		} `json:"files"`
+		MetaData []interface{} `json:"meta_data"`
```

Comment:
Also, you are parsing the `MetaData`, but never printing it

Reply:
Same for the `Signed` field I think.

Reply:
Could you tell more more about MetaData? `meta_data is an object of unknown structure as this is dependent of update type (also custom defined by user)`
I would introduce a detail level 4 to enable printing this `meta_data`. 

`Signed` is printed here https://github.com/siredmar/mender-cli/blob/5ea07ab72cb86e1e7220d7799dc9de5dbd3072b4/client/deployments/client.go#L140

Reply:
Sorry, missed the Signed part (sausage fingers).

The `MetaData` is just a generic JSON structure, with not requirements on the structure, as far as Mender is concerned. Hence, this can be whatever, as long as it is JSON. I guess this can be printed in `level 4` as  you suggest, with something along the lines of:
```
fmt.Printf("%+v\n", metaData)
```
or something simlar like: `%#v` as the verb maybe. Experiment a little, and see which you prefer :)

Reply:
sticking with `fmt.Printf("  MetaData: %v\n", v.MetaData)`